Use helm for installation of service-broker-proxy

fabianschwarzfritz opened the pull request:
Unfortunately, the pull request #9  by @mvladev was automatically closed.
With this pull request I have cherry-picked @mvladev's commit so that we can review it here.

Thanks @mvladev again for your contribution.

hummelo commented on charts/service-broker-proxy/templates/service.yaml:
```diff
@@ -0,0 +1,19 @@
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: http
```
port needs to be changed

hummelo commented on README.md:
````diff
@@ -6,39 +6,42 @@
-EOF
+```bash
+helm install charts/service-broker-proxy --name service-broker-proxy --namespace service-broker-proxy
+# --set image.repository=fooo-repository # optional override of proxy's image
````
overwrite not override

fabianschwarzfritz commented:
Hi @hummelo .

I have done the requested improvements:
 - fixed the *targetPort*, a `kubectl get endpoints` now shows the correct endpoint. 
 - "override" is actually the correct expression in the README
 - I have added example values to the README on how to configure the service-broker-proxy-k8s deployment. 

Regards,
Fabian
